pom.xml

```diff
@@ -135,8 +135,8 @@ THE SOFTWARE.
   </scm>
 
   <properties>
-    <jenkins.version>2.7.3</jenkins.version>
-    <java.level>7</java.level>
+    <jenkins.version>2.54</jenkins.version>
```

Comment:
Better to use 2.60.x LTS then

Reply:
I changed to the minimum version requiring java 8

Reply:
I think Oleg's point is that it is generally preferred to use LTS versions for baselines, and 2.60 is the first LTS that required Java 8.

Reply:
I will change to the LTS then